dts/arm/st/stm32f429.dtsi

```diff
@@ -25,5 +25,15 @@
 			status = "disabled";
 			label = "UART_8";
 		};
+
+		spi4: spi@40013400 {
+			compatible = "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40013400 0x400>;
+			interrupts = <84 5>;
+			status = "disabled";
+			label = "SPI_4";
+		};
```

Comment:
What about SPI5 and SPI6?

Reply:
done and added comment and example stm32f429vX.dtsi for /delete-node/